src/JuMP.jl

```diff
@@ -738,5 +729,26 @@ include("print.jl")
 # Nonlinear-specific code
 include("nlp.jl")
 
+
+typealias _getTypes Union{NonlinearExpression,NonlinearParameter, AffExpr}
+function getValue{T<:_getTypes}(arr::Array{T})
```

Comment:
Sure, I'll do that. 

Out of interest, what are the changes in `map` that are coming up (reference)?